protobufs/gen/swagger/entity/service.swagger.json

```diff
@@ -98,6 +98,38 @@
         ]
       }
     },
+    "/entity/{identifier}/relationship/{relationship_identifier}": {
+      "get": {
+        "summary": "Entity Relation Get",
+        "description": "Get entity from business partner",
+        "operationId": "GetEntityByRelationship",
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/entityEntityResponse"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "name": "identifier",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "relationship_identifier",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "tags": [
+          "DocumentService"
```

Comment:
we could rename this in the corresponding proto file to EntityService, as per the housekeeping discussion yesterday